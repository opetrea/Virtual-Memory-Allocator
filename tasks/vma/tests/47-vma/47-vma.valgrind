==92284== Memcheck, a memory error detector
==92284== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==92284== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==92284== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==92284== Parent PID: 92283
==92284== 
==92284== 
==92284== HEAP SUMMARY:
==92284==     in use at exit: 0 bytes in 0 blocks
==92284==   total heap usage: 13 allocs, 13 frees, 8,433 bytes allocated
==92284== 
==92284== All heap blocks were freed -- no leaks are possible
==92284== 
==92284== For lists of detected and suppressed errors, rerun with: -s
==92284== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
