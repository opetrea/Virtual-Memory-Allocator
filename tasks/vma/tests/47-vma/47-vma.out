Invalid permissions for read.
gigel
